ddsketch/store/store.go

```diff
@@ -26,6 +26,7 @@ type Store interface {
 	AddBin(bin Bin)
 	AddWithCount(index int, count float64)
 	Bins() <-chan Bin
+	ForEach(f func(b Bin))
```

Comment:
Can we make it `ForEach(f func(b Bin) bool)` to allow early stopping? E.g., we iterate while `true` is returned.